python/agents/image-scoring/image_scoring/sub_agents/scoring/tools/get_images_tool.py

```
@@ -0,0 +1,22 @@
+from google.adk.tools import ToolContext
+
+
+async def get_image(tool_context: ToolContext):
+    try:
+        
+        artifact_name = (
+            f"generated_image_" + str(tool_context.state.get("loop_iteration", 0)) + ".png"
+        )
+        artifact = await tool_context.load_artifact(artifact_name)
```

Ruff (0.11.9)

Local variable `artifact` is assigned to but never used

Remove assignment to unused variable `artifact`

(F841)